pkg/sql/opt/optbuilder/create_function.go

```diff
@@ -148,15 +148,10 @@ func (b *Builder) buildCreateFunction(cf *tree.CreateFunction, inScope *scope) (
 	outScope = b.allocScope()
 	outScope.expr = b.factory.ConstructCreateFunction(
 		&memo.CreateFunctionPrivate{
-			Schema:       schID,
-			FunctionName: &funcName,
-			Replace:      cf.Replace,
-			Args:         cf.Args,
-			ReturnType:   cf.ReturnType,
-			Options:      funcOptions,
-			Body:         tree.FunctionBodyStr(fmtCtx.String()),
```

Comment:
oh...looks like you need to override the function body....

Reply:
Do we need to format the body? Can't we just leave it as the user specified it?

Reply:
Oh. We need to do this to make sure the references are fully qualified?

Reply:
yes correct.

Reply:
Fixed.